lib/ebaze_web/controllers/session_controller.ex

```diff
@@ -1,16 +1,19 @@
 defmodule EbazeWeb.SessionController do
   use EbazeWeb, :controller
 
-  alias Ebaze.Repo
   alias Ebaze.{Accounts, Accounts.User}
 
   def new(conn, _) do
-    changeset = Accounts.change_user(%User{})
-
-    render(conn, "new.html",
-      changeset: Accounts.change_user(%User{}),
-      action: Routes.session_path(conn, :create)
-    )
+    if get_session(conn, :cookies) do
```

Comment:
So this only checks if the session is present—can this be 'faked'? If you create a fake cookie manually , can you get in?

Reply:
an option would be to check for a response cookie from the http request

Reply:
Well, you can set the cookies in the browser...can you make it so the app cookies are signed/encrypted? That they hold say a ID or something the app knows is legit?

Also: we have used https://github.com/ueberauth/guardian for another project